docs/source/contributing.rst

```diff
@@ -138,6 +138,27 @@ Documentation
 
 We use `numpydoc <http://numpydoc.readthedocs.io/en/latest/format.html>`_ for our docstrings.
 
+Building the docs is possible with
+
+.. code-block:: none
+
+   $ conda env create -f ci/environment-docs.yaml --name=dask-ml-dev-docs
+
+This will build the full documentation after a complete installation. For a
```

Comment:
I'd prefer to just remove the remainder of this. I'm in the process of migrating all our examples to examples.dask.org, and removing ones that aren't relevant.

Reply:
Removed.